app/views/home/_topbar.html.erb

```diff
@@ -41,6 +41,14 @@
           <%= render_menu :top_menu %>
         </ul>
         <ul class="nav navbar-nav navbar-admin navbar-right navbar-collapse collapse" id="menu2">
+          <% if Setting[:experimental_ui]%>
+              <li class="dropdown menu_tab_dropdown" id="react_root">
```

Comment:
Why does this need `id="react_root"`?

Reply:
Because that's the ID we will reference in order to mount the react application.  See https://github.com/Katello/katello/pull/6992/files#diff-52f1696d7dae32f43202243e041b3af8R7 for an example.

Reply:
are we expecting only one? in that case we should probably call id="experimental_ui" instead of react.

Reply:
Changed the name of the ID.  What do you mean by "only one"?  Only one what?